linkerd/app/outbound/src/metrics/transport.rs

```diff
@@ -211,7 +211,11 @@ where
 
     #[inline]
     fn poll_ready(&mut self, cx: &mut Context<'_>) -> Poll<Result<(), Self::Error>> {
-        self.inner.poll_ready(cx).map_err(Into::into)
+        self.inner.poll_ready(cx).map_err(|error| {
+            let error = error.into();
+            self.metrics.inc_closed(Some(&*error));
```

Comment:
I wonder whether we should only increment the closed counter or we should increment both the open and the close ones.. 🤔